distributed/shuffle/_worker_extension.py

```diff
@@ -240,7 +240,6 @@ async def get_output_partition(
         """Get an output partition to the shuffle run"""
 
 
-# TODO remove quotes on tuple (requires Python >=3.9)
```

Comment:
I don't see any other changes in this file. Not sure what tuple this comment was referring to. Maybe this was a stale comment?

Reply:
I'm pretty sure I refactored whatever was there before.

Reply:
yeah I think it's stale, the only quotes are around "np.array" which still needs them